index.html

```diff
@@ -0,0 +1,201 @@
+            </div>
+            <div class="date">
+                <h3 class="date-head">2015.10.15(THU) ~ 16(FRI)</h3>
+                <p class="date-second">@ National Museum of Karea, Art Center Nabi and more</p>
+            </div>
+        </div>
+    </section>
+    <section class="main-program">
+        <div class="main-program-container">
+           <div class="main-heading">
+            <h3>Main program</h3>
+            <hr>
+           </div>
+           <div class="main-program-content">
+            <div class="contents">
+                <img src="/images/program_icon_01 1.svg">
+                <h4>Lecture</h4>
+                <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>
+            <div class="contents">
+                <img src="/images/program_icon_02 1.svg">
+                <h4>Forum</h4>
+                <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>
+            <div class="contents">
+                <img src="/images/program_icon_03 1.svg">
+                <h4>CC Exhibition</h4>
+                <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>
+            <div class="contents">
+                <img src="/images/program_icon_04 1 (1).svg">
+                <h4>Workshop</h4>
+                <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>
+            <div class="contents">
+                <img src="/images/program_icon_05 1.svg">
+                <h4>CC Ignite</h4>
+                <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>
+            <div class="main-btn none">
+                <a>Join cc global submit 2015</a>
+            </div>
+           </div>
+        <div class="remove see-all">
+            <h3>SEE THE WHOLE PROGRAM</h3>
+        </div>
+        </div>
+    </section>
+    <section class="featured">
+        <div class="featured-title">
+            <h3>Featured speakers</h3>
+            <hr>
+        </div>
+        <div class="featured-container">
+        </div>
+        <ul class="featured-btn none">
+            <li class="see-more none"><a href="#">MORE</a>
+            <i class="fa fa-chevron-down"></i></li>
+        </ul>
+    </section>
+    <section class="partner remove ">
+        <div class="partner-container">
+            <h3 class="partner-heading">Partner</h3>
+            <hr>
+        </div>
+        <div class="companies">
+            <img src="/images/mozilla 1.png">
+            <img src="/images/googlekorea 1.png">
+            <img src="/images/naver 1.png">
+            <img src="/images/daumkakaologo 1.png">
+            <img src="/images/airbnb 1.png">
+        </div>
+    </section>
```

Comment:
- [ ] Please apply more spacing to the sides of this section and reduce the space between the partners to match the design.

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/olanikegloria/capstone-project/assets/62290545/587c77a7-973f-4ab3-83ac-7676ec3605a1) | ![image](https://github.com/olanikegloria/capstone-project/assets/62290545/26345361-c87e-4613-b702-844d9ac43de3) |